lib/auth/join_github.go

```diff
@@ -27,7 +27,7 @@ import (
 )
 
 type ghaIDTokenValidator interface {
-	Validate(context.Context, string) (*githubactions.IDTokenClaims, error)
+	Validate(context.Context, string, string) (*githubactions.IDTokenClaims, error)
```

Comment:
might be more clear to name the args here, I'd find it helpful at least

Reply:
Agreed.

Reply:
cc70c337ef